sdk/deploy.js

```diff
@@ -75,6 +75,7 @@ const deployConf = async function deployConf(accountId, funcName, apiKey, funcCo
       if (env[key] == null) {
         env[key] = process.env[key];
         if (!env[key] || env[key] === '') {
+          logger.warn(`Ignoring empty env ${key}`);
```

Comment:
Then how do I pass an empty value?

I'd understand warning `if (! key in process.env)`, but I *should be* allowed explicitly to specify an empty value for a variable.

Reply:
Thanks!